src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -75,7 +77,7 @@ HttpRequestMessage RequestFunc()
             return stringResponse;
         }
 
-        public Task SaveStateTransactionallyAsync(string actorType, string actorId, string data, CancellationToken cancellationToken = default)
+        public Task SaveStateTransactionallyAsync(string actorType, string actorId, string data, CancellationToken cancellationToken = default, List<Autogenerated.TransactionalActorStateOperation> grpcData = default)
```

Comment:
Is `grpcData` used?

Reply:
Yes the grpcInteractor uses this for its data. It doesnt use the string argument